### ISSUE TITLE
A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_Keyboard: Image present under ‘Database Size’ section isn’t accessible using keyboard and alt attribute is missing for it

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9267]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.” 

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOS;#MAS2.1.1;#MAS1.1.1;#Keyboard; 

**Environment Details:** 
Application Name: Azure Data Studio 
Version: 1.15.0-insider 
Commit: 3a8b74a 
Date: 2020-02-11T05:49:41.181Z 
VS Code: 1.42.0 
Electron: 7.1.11 
Chrome: 78.0.3904.130 
Node.js: 12.8.1 
V8: 7.8.279.23-electron.0 
OS: Darwin x64 19.3.0 
Mac OS:10.15.3 
 
Additional Details: 
Mac OS:10.15.3 
MAS References: MAS2.1.1, MAS1.1.1 

**Repro Steps:** 
1. Launch Azure Data Studio. 
2. Connect to server. 
3. In the Connections pane, right-click on a server & select Manage. 
4. Launch Voiceover 
5. Use Control +Option +right arrow key and try to navigate to graphical image present under Database Size section.  

**Actual:** 
**Issue 1:** Alt attribute is not present for the graphical image.  
**Issue 2:** When hovered mouse over any strip of pink/blue colour, tooltip is displayed on screen, but the same is not accessible using keyboard.  

Same issue is observed in Database Dashboard screen: 1. Launch Azure Data Studio.2. Connect to server.3. In the Connections pane, navigate to server --> database --> right click on any database & select Manage. 

**Expected:** 
**Issue 1:** Alt attribute should be present for the graphical image, so that while voiceover is on and when navigated to it using Control +Option +right arrow key, screenreader should announce description of the image. 
**Issue 2:** Tooltips of any strip of pink/blue colour should be accessible using keyboard.  
 
**User Impact:** 
Keyboard and Screen reader users will face difficulty to understand exactly what content is present under ‘Database Size’ section, as graph is neither accessible by keyboard nor it provides any alt description for screen reader users. 

**Attachment link for Reference:**  
[9267_A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_Keyboard.zip](https://github.com/microsoft/azuredatastudio/files/4274857/9267_A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_Keyboard.zip)

**Comment by alanrenmsft** [comment]
by design? i don't think we need to have accessibility support for charts.

**Comment by alanrenmsft** [comment]
we've reviewed this long time ago, and we don't support keyboard navigation for charts.

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
